dbld/images/helpers/functions.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Helper functions for container creation.
+
+function add_obs_repo_debian {
+    DISTRO=$1
+    wget -qO - http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/${DISTRO}/Release.key | apt-key add -
+    echo "deb http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/"${DISTRO}" ./" | tee --append /etc/apt/sources.list.d/syslog-ng-obs.list
+    apt-get update -qq
+}
+
+function add_czanik_repo_centos6 {
+    cd /etc/yum.repos.d
+    wget https://copr.fedorainfracloud.org/coprs/czanik/syslog-ng39epel6/repo/epel-6/czanik-syslog-ng39epel6-epel-6.repo
```

Comment:
I'm not sure if we can do anything about this but it is unfortunate to have syslog-ng 3.9 to be hardcoded here.